src/components/home/elements/SendToken.vue

```diff
@@ -904,6 +946,11 @@ export default {
         },
         formatter(num) {
             return browser.bigNumberFormatter(num)
+        },
+        getAlias() {
+            if (JSON.parse(window.localStorage.getItem(this.defaultAddress)).hasOwnProperty('alias')) {
```

Comment:
if no `alias`, the method should return empty dictionary.
And `(JSON.parse(window.localStorage.getItem(this.defaultAddress))` is too long. To simply the code and make code readable, I suggest add local variable here.
```
let userData = (JSON.parse(window.localStorage.getItem(this.defaultAddress))
if (userData.hasOwnProperty('alias')) {
    return userData.alias
} else {
    return {}
}
```

Reply:
Also apply the same method in `Send.vue`.

Reply:
Still getting error in console. We need check `userData` is null or not at first. 
```
if (userData && userData.hasOwnProperty('alias')) {
   ...
}
```
<img width="673" alt="Screenshot 2020-04-17 at 1 45 20 PM" src="https://user-images.githubusercontent.com/41148850/79536134-d89dac00-80b1-11ea-881c-8a42516adde5.png">